Don't fetch the Unicode conformance data over the network at configure time:
file(DOWNLOAD) breaks under FEATURES=network-sandbox. The two files are supplied
by app-i18n/unicode-data (BDEPEND, test?) and copied into the source root by the
ebuild; copy them into the build tree's data/ dir where the
testgraphemetest/testnormtest cases expect them.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -67,8 +67,8 @@ if(UTF8PROC_ENABLE_TESTING)
   enable_testing()
   file(MAKE_DIRECTORY data)
   set(UNICODE_VERSION 16.0.0)
-  file(DOWNLOAD https://www.unicode.org/Public/${UNICODE_VERSION}/ucd/NormalizationTest.txt ${CMAKE_BINARY_DIR}/data/NormalizationTest.txt SHOW_PROGRESS)
-  file(DOWNLOAD https://www.unicode.org/Public/${UNICODE_VERSION}/ucd/auxiliary/GraphemeBreakTest.txt ${CMAKE_BINARY_DIR}/data/GraphemeBreakTest.txt SHOW_PROGRESS)
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/NormalizationTest.txt ${CMAKE_BINARY_DIR}/data/NormalizationTest.txt COPYONLY)
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/GraphemeBreakTest.txt ${CMAKE_BINARY_DIR}/data/GraphemeBreakTest.txt COPYONLY)
   add_executable(case test/tests.h test/tests.c utf8proc.h test/case.c)
   target_link_libraries(case utf8proc)
   add_executable(custom test/tests.h test/tests.c utf8proc.h test/custom.c)
